packages/esm-appointments-app/src/patient-appointments/patient-appointments-header.tsx

```diff
@@ -0,0 +1,40 @@
+import { useTranslation } from 'react-i18next';
+import React from 'react';
+import { ArrowLeft } from '@carbon/react/icons';
+import { Button } from '@carbon/react';
+import { useNavigate } from 'react-router-dom';
+import styles from './patient-appointments-header.scss';
+import { PatientBannerPatientInfo, PatientPhoto } from '@openmrs/esm-framework';
+
+interface PatientAppointmentsHeaderProps {
+  patient: fhir.Patient;
+}
+
+const PatientAppointmentsHeader: React.FC<PatientAppointmentsHeaderProps> = ({ patient }) => {
+  const { t } = useTranslation();
+  const navigate = useNavigate();
+  const patientName = `${patient?.name?.[0]?.given?.join(' ')} ${patient?.name?.[0].family}`;
```

Comment:
Not sure if this is the *best* way to do this, but this is what we were doing in the example in the patient chart, so wanted to keep it consistent for now.  (This is just used to render the mouseover on the patient image)